### ISSUE TITLE
Update "react-sortable-hoc" to avoid incorrect peer dependency

### ISSUE THREAD

**Issue by mhouel** [NadavShaar/react-grid-table#70]
Hi,

The project has a peer dependency with React 17, and a dependency with React Grid Table 1.1.
That plugin has a peer dependency with React 14 || 15 || 16, and thus, trigger an incorrect peer dependency warning :

```
warning "@nadavshaar/react-grid-table > react-sortable-hoc@1.11.0" has incorrect peer dependency "react@^0.14.0 || ^15.0.0 || ^16.0.0".
warning "@nadavshaar/react-grid-table > react-sortable-hoc@1.11.0" has incorrect peer dependency "react-dom@^0.14.0 || ^15.0.0 || ^16.0.0".
```

They published a 2.0 version just 20 days ago, compatible with React 17.
Would it be possible to update the dependency ? or maybe change the project `peerDependencies` ?
Thanks !

**Comment by NadavShaar** [comment]
Thanks a lot, will update the package soon.

**Comment by NadavShaar** [comment]
Fixed in version 0.3.0
